.github/workflows/pre-commit.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - uses: actions/setup-python@v2
       - uses: actions/setup-go@v2
         with:
-          go-version: '^1.18.0'
```

Comment:
suggest not changing this

Reply:
abhishek has created a ticket to track this: https://github.com/ObolNetwork/charon/issues/945
1.18 in precommit workflow was attempting to modify files with version 1.19: https://github.com/ObolNetwork/charon/runs/7745788084?check_suite_focus=true